static/js/around-map.js

```diff
@@ -7,34 +7,33 @@ const map = new mapboxgl.Map({
 })
 
 // users interesting points
-const pois = new Worker('/static/js/webworker-around.js')
+const worker = new Worker('/static/js/webworker-around.js')
 
-pois.onmessage = informations => {
-  const [name, wifi, power, comment, lng, lat, id] = informations.data
-  const el = document.createElement('div')
-  el.classList.add('marker')
+worker.onmessage = response => {
+  response.data.forEach(poi => {
+    const el = document.createElement('div')
+    el.classList.add('marker')
 
-  let content = `<h2>Hello ${name}</h2>
-                 <ul>
-                    <li>Wifi quality: ${wifi}</li>
-                    <li>Power available: ${power}</li>
-                    <li>Comments: ${comment}</li>
-                  </ul>
-                  <button value=${id} onclick=sharePosition(value)>share this position</button>`
-
-  const popup = new mapboxgl.Popup({offset: [10, -20]})
-    .setHTML(content)
+    const popup = new mapboxgl.Popup({offset: [10, -20]})
+      .setHTML(`<h2>Hello ${poi.name}</h2>
+                   <ul>
+                      <li>Wifi quality: ${poi.wifiQuality}</li>
+                      <li>Power available: ${poi.powerAvailable}</li>
+                    </ul>
+                    <button value=${poi._id} onclick=sharePosition(value)>share this position</button>`
+              )
 
-  new mapboxgl.Marker(el, {offset:[0, -30]})
-    .setLngLat([lng, lat])
-    .setPopup(popup)
-    .addTo(map)
-};
+    new mapboxgl.Marker(el, {offset:[0, -30]})
+      .setLngLat([poi.position.longitude, poi.position.latitude])
+      .setPopup(popup)
+      .addTo(map)
+  })
+}
 
 pois.postMessage('info-requested')
```

Comment:
if you create `const worker = new Worker('...')`, you should use `worker.postMessage('...')` isn't ? 

I changed it ;) that was to do a review to you ! mouhahahahahahahahah